code/game/gamemodes/devil/true_devil/_true_devil.dm

```diff
@@ -0,0 +1,202 @@
+#define DEVIL_R_HAND_LAYER 1
+#define DEVIL_L_HAND_LAYER 2
+#define DEVIL_TOTAL_LAYERS 2
+
+
+/mob/living/carbon/true_devil
+	name = "True Devil"
+	desc = "A pile of infernal energy, taking a vaguely humanoid form."
+	icon = 'icons/mob/32x64.dmi'
+	icon_state = "true_devil"
+	gender = NEUTER
+	health = 350
+	maxHealth = 350
+	ventcrawler = 0
+	density = 1
+	pass_flags =  0
+	var/ascended = 0
+	sight = (SEE_TURFS | SEE_OBJS)
+	status_flags = CANPUSH
+	universal_understand = 1
+	universal_speak = 1 //The devil speaks all languages meme
+	var/mob/living/oldform
+	var/list/devil_overlays[DEVIL_TOTAL_LAYERS]
+
+/mob/living/carbon/true_devil/New()
+	var/obj/item/organ/internal/brain/B = new(src)
+	B.insert()
+	..()
+
+
+/mob/living/carbon/true_devil/proc/convert_to_archdevil()
+	maxHealth = 5000 // not an IMPOSSIBLE amount, but still near impossible.
+	ascended = 1
+	health = maxHealth
+	icon_state = "arch_devil"
+
+/mob/living/carbon/true_devil/proc/set_name()
+	name = mind.devilinfo.truename
+	real_name = name
+
+/mob/living/carbon/true_devil/Login()
+	..()
+	mind.announceDevilLaws()
+	var/obj_count = 1
+	to_chat(src, "\blue Your current objectives:")
+	for(var/datum/objective/objective in mind.objectives)
+		to_chat(src, "<B>Objective #[obj_count]</B>: [objective.explanation_text]")
+		obj_count++
+
+
+/mob/living/carbon/true_devil/death(gibbed)
+	stat = DEAD
+	..(gibbed)
+	drop_l_hand()
+	drop_r_hand()
+	spawn (0)
+		mind.devilinfo.beginResurrectionCheck(src)
+
+
+/mob/living/carbon/true_devil/examine(mob/user)
+	var/msg = "<span class='info'>*---------*\nThis is \icon[src] <b>[src]</b>!\n"
```

Comment:
`var/list/msg` = list(text)`
`msg += text` will work fine as before
Then at the end, do `msg.Join("\n")` to get the text split by the newline